src/Facebook/InstantArticles/Transformer/Rules/InstantArticleRule.php

```diff
@@ -80,7 +80,7 @@ public function apply($transformer, $instant_article, $node)
             $instant_article->disableAutomaticAdPlacement();
         } else {
             $instant_article->enableAutomaticAdPlacement();
-            $pairs = explode(' ', $auto_ad_placement, 2);
+            $pairs = explode(' ', $auto_ad_placement ?: '', 2);
```

Comment:
Is this intentional?
`$auto_ad_placement ? $auto_ad_placement : ''`

Reply:
yes, just avoiding passing null to explode as now it's typed and it doesn't allow it.